vulcanai/tests/plotters/test_visualization.py

```diff
@@ -168,6 +172,7 @@ def test_display_pca(self, dnn_class):
 
     def test_display_tsne(self, dnn_class):
         """Test t-SNE displays and saves."""
+        warnings.filterwarnings('ignore', category=PendingDeprecationWarning)
```

Comment:
can we instead look for something to replace this that's not pending deprecation? we may find ourselves in trouble in the future and may as well figure out how to solve it now.

Reply:
My comment above is still relevant ^ I believe.